### ISSUE TITLE
crossfade() ignores liq_fade_out

### ISSUE THREAD

**Issue by RM-FM** [savonet/liquidsoap#1041]
Hi,

For some reason fade outs are not working with below setup. Instead there is a hard cut.

```
s = request.dynamic(get_request)	
s = cue_cut(s)
s = crossfade(s)
```

Cue- and Fade Points are provided using meta data:
`liq_cue_in=0.24,liq_cue_out=214.11,liq_cross_duration=4.52,liq_fade_in=0.0,liq_fade_out=4.52`

**Version details**
 - Ubuntu 18.04
 - 1.4.0 (opam)

**Comment by RM-FM** [comment]
@toots Were you already in a position to check this one?

**Comment by toots** [comment]
Hello! I've been busy. I'll look at it asap.

**Comment by toots** [comment]
Hi,

I was working on #1009 and noticed a bunch of issues with the new fades included in the `1.4.0` release. I'll work on them asap. This will most likely fix your issue as well.
